src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -186,7 +194,7 @@ Shortcut ShortcutsRegister::readShortcut(framework::XmlReader& reader) const
         } else if (tag == STANDARD_KEY_TAG) {
             shortcut.standardKey = QKeySequence::StandardKey(reader.readInt());
         } else if (tag == SEQUENCE_TAG) {
-            shortcut.sequence = reader.readString();
+            shortcut.sequence += reader.readString() + "\n";
```

Comment:
\n can never possibly be part of a shortcut sequence name so this always safe, even if it a very slight kludge

Reply:
Could you please explain the purpose of this change?

Reply:
Currently it's ignoring multiple sequences for the same action (e.g. Dell or backspace to mean "delete")